### ISSUE TITLE
Could not load type "formextra_recaptcha"

### ISSUE THREAD

**Issue by sunviwo** [simplethings/SimpleThingsFormExtraBundle#46]
I get this error and I don't know why. Any help ?

**Comment by stof** [comment]
have you enabled the recaptcha in the bundle, by configuring it ?

**Comment by sunviwo** [comment]
Yes, I followed the documentation and here is what I put in my config.yml
simple_things_form_extra:  
    recaptcha:
        public_key: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
        private_key: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
